src/components/v5/shared/SearchSelect/partials/SearchItem/SearchItem.tsx

```diff
@@ -39,13 +37,14 @@ const SearchItem: FC<SearchItemProps> = ({
           showAvatar,
           color,
           missingPermissions,
+          walletAddress,
           token,
         }) => {
           const firstDisabledOption = options.filter(
             (option) => option.isDisabled,
           )[0];
           const labelText =
-            typeof label === 'string' ? label : formatMessage(label);
+            typeof label === 'string' ? label : formatText(label || '');
```

Comment:
I think you do not need to check if label is string you can just pass label to `formatText`